app/channel.go

```diff
@@ -340,6 +340,30 @@ func (a *App) createGroupChannel(userIds []string, creatorId string) (*model.Cha
 	}
 }
 
+func (a *App) GetGroupChannel(userIds []string) (*model.Channel, *model.AppError) {
+	if len(userIds) > model.CHANNEL_GROUP_MAX_USERS || len(userIds) < model.CHANNEL_GROUP_MIN_USERS {
```

Comment:
Do we need this check?

Reply:
Ok, my point here is, if you can't create a GroupChannel with that limits, the channel wont exists anyway, so, you can try to get it and will fail as "not-exists" more proper error than, "you can't create it"

Reply:
OK. However, given we then fetch all the users requested from the database, I'd prefer to filter out excess requests before we hit the database with them. Also, although the error "key" mentions `create`, the actual string is suitably generic for this case as well.